docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
+After the run completes, you can:
+
+- Click the **asset** to open a sidebar containing info about the asset, including its last materialization stats and a link to view the [**Asset details** page](/concepts/dagit/dagit#asset-details)
+- Click the ID of the **Latest Run** - in the above image, that's `651489a2` - in an asset to view the [**Run details**](/concepts/dagit/dagit#run-details) page. This page contains detailed info about the run, including timing information, errors, and logs.
 
 ---
 
-## Step 3: View the assets in the Dagster UI
+## Step 4: Review the Python code in your Dagster project
 
-In this step you'll start Dagster's web UI.
+You saw how you can create a Dagster project that loads a dbt project. How does this work? Understanding how Dagster loads a dbt project will give you a foundation for customizing how Dagster runs your dbt project, as well as for connecting it to other data assets outside of dbt.
 
-1. To start the UI, run the following in `/tutorial_template`:
+The most important file is the Python file that contains the set of definitions for Dagster to load: `jaffle_shop/definitions.py`. Dagster executes the code in this file to find out what assets it should be aware of, as well as details about those assets. For example, when you ran `dagster dev` in the previous step, Dagster executed the code in this file to determine what assets to display in the UI.
 
-   ```shell
-   dagster dev
-   ```
+In order to return a Dagster asset for each dbt model, the code in this file needs to find out what dbt models you have. It finds that out by reading a file called a "manifest.json", which is a file that dbt can generate for any dbt project and contains information about every model, seed, snapshot, test, etc. in the project. Here's the code that does this:
 
-   Which will result in output similar to:
 
-   ```shell
-   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
-   ```
+```python
+dbt_project_dir = Path(__file__).parent.joinpath("..", "..")
+dbt = DbtCliResource(project_dir=os.fspath(dbt_project_dir))
+
+# If DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is set, a manifest will be created at runtime.
+# Otherwise, we expect a manifest to be present in the project's target directory.
+if os.getenv("DAGSTER_DBT_PARSE_PROJECT_ON_LOAD"):
+    dbt_parse_invocation = dbt.cli(["parse"], manifest={}).wait()
+    dbt_manifest_path = dbt_parse_invocation.target_path.joinpath("manifest.json")
+else:
+    dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
+```
 
-2. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
+As the comment explains, the code gives you a choice about how to create this manifest.json file. Based on an environment variable, either:
+1. This code generates the manifest.json for you.
+2. This code leaves it up to you to generate the manifest.json file on your own, and this code just reads it.
```

Comment:
```suggestion
As the comment explains, the code gives you a choice about how to create this `manifest.json file`. Based on an environment variable, either:

1. This code generates the `manifest.json` for you.
2. This code leaves it up to you to generate the `manifest.json` file on your own, and this code reads it.
```

Adding formatting for `manifest.json` and a line break for readability.

Should we call out the environment variable specifically here? Ex:

> Based on the `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD` environment variable, either: